flake.nix

```diff
@@ -71,6 +71,23 @@
 
           pkgs = import nixpkgs { inherit system; };
 
+          # Download statically linked vmtest binary
```

Comment:
Maybe a question not necessarily related to this PR but is it worth have separate nix flakes depending on the kind of environment we want e.g. in my local development I don't need these packages unless I want to run a vm test right?

Reply:
Possibly - let's wait longer and see. Right now anything that could possibly be used by CI is included in flake. I think there's a lot of value in keeping things consistent across CI and developer env.  But if it gets too heavy in the long term we can reconsider how we handle this.